provision/provision.go

```diff
@@ -511,9 +519,6 @@ func (s AutoScaleSpec) ToCPUValue(a App) (int, error) {
 }
 
 func (s AutoScaleSpec) Validate(quotaLimit int, a App) error {
-	if s.MinUnits == 0 {
```

Comment:
@crgisch why you removed this validation?

Reply:
the ideia here was to allow setting min: 0 in order to allow schedule to downscale apps to 0